test_verbosefilter.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, message_list):
         self.message_list = message_list
 
     def emit(self, record):
-        self.message_list.append(record.msg)
+        self.message_list.append(record.getMessage())
```

Comment:
This change is necessary to get `test_reformatting` to fail with the old code. It yields the same TypeError. With the new code, of course, it passes.